recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-h3ulcb-4x2g-ab-domd.dts

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * Device Tree Source for the H3ULCB Aosbox (CCPF Starter kit) board
+ * running XEN hypervisor
+ *
+ * Copyright (C) 2021 Renesas Electronics Corp.
```

Comment:
Here and elsewhere: do we need to put EPAM's copyright as well?

Reply:
added copyright